W-14414684 Updating tutorial (#194)
mulesoft/docs-general: modules/ROOT/pages/api-led-design.adoc

* Explore the security requirements.

After you define the scope and flow of your integration project,

> define an API specification in link:https://raml.org/[RAML] or link:https://www.openapis.org/[OAS].

Reviewer: According to ACB, OAS is (way) more prevalent, so maybe put it first?

Author: RAML is MuleSoft.  :-)

Reviewer: Yes, I know.

. Open API Designer:
+
link:https://anypoint.mulesoft.com/designcenter/#/projects["Take me to API Designer^", role="button-primary"]

> . Select *Create + > New API Specification* to open the API Designer editor.

Reviewer: Again, I think the punctuation in the menus could be dropped.

. Click *GET* to display the GET request and more information about the specification.
+
image:code-response1.png[Response field, 500]
. Click *Show* to review code examples for each protocol.
. Click the *200* and *404* tabs to review the responses defined in the API specification.

> . Click the *Try it* button.

Reviewer: ```suggestion
. Click the *Try it* button:
```

+
image:try-it.png[The Try It button, 500]
. Click *Send* to send your request to the temporary request URL that the mocking service creates from your specification.
+
It's safe to ignore any error messages on this screen. A successful request returns `200 OK` and the test message:
+
image:successful-test1.png[Results of a successful test, 500]

> . Select *Response details* in the more menu to examine the response headers and request headers in the mocking service to help diagnose issues or understand the behavior of your API specification.

Reviewer: If "more menu" is a kebab...
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-ellipsismenu

> . When you have finished testing, open the *Mocking Service Configuration* panel.

Reviewer: ```suggestion
. When you finish testing, open the *Mocking Service Configuration* panel.
```

. In *Local Settings*, enable *Select By Default*.
+
image:api-mocking-service-configuration-icon.png[Mocking service icon]
+
image:disable-mocking-service.png[Mocking service control before being disabled, 500]

== Step 2.4. Publish Your API Specification

After testing your API, publish it to your private Exchange so others in your organization can reuse your work.

. Open `hello-world-api.raml` if it isn't already open:
+
link:https://anypoint.mulesoft.com/designcenter/#/projects["Take me to API Designer^" role="button-primary"]
. Click *Publish*.

> . In the Publishing to Exchange dialog window, accept the defaults and click *Publish to Exchange*.

Reviewer: ```suggestion
. In the *Publishing to Exchange* dialog, accept the defaults and click *Publish to Exchange*:
```

+
image:publish-to-exchange1.png[User interface for publishing to Exchange, 500]
+
. Click *Close* in the confirmation dialog.

> After publication, anyone in your organization can see the hello-world-api API specification and reuse it.

Reviewer: ```suggestion
After publication, anyone in your organization can see the `hello-world-api` API specification and reuse it.
```
